classes/UpgradeTools/CoreUpgrader/CoreUpgrader.php

```diff
@@ -488,19 +495,10 @@ private function hasPhpError($phpRes): bool
         return isset($phpRes) && (is_array($phpRes) && !empty($phpRes['error'])) || $phpRes === false;
     }
 
-    /**
-     * @param string $upgrade_file
-     * @param string $query
-     * @param mixed $phpRes
-     *
-     * @return void
-     */
-    private function logPhpError(string $upgrade_file, string $query, $phpRes): void
+    private function logPhpError(string $upgrade_file, string $query, string $message): void
     {
         $this->logger->error(
-            '[ERROR] PHP ' . $upgrade_file . ' ' . $query . "\n" .
-            (empty($phpRes['error']) ? '' : $phpRes['error'] . "\n") .
-            (empty($phpRes['msg']) ? '' : ' - ' . $phpRes['msg'] . "\n")
+            '[ERROR] PHP ' . $upgrade_file . ' ' . $query . "\n" . $message
```

Comment:
I would take the opportinity to remove the [ERROR] from the original string. The logger is likely able to add it